docs/content/docs/dev/configuration/overview.md

```diff
@@ -0,0 +1,219 @@
+---
+title: "Overview"
+weight: 1
+type: docs
+aliases:
+- /dev/project-configuration.html
+- /start/dependencies.html
+- /getting-started/project-setup/dependencies.html
+- /quickstart/java_api_quickstart.html
+- /dev/projectsetup/java_api_quickstart.html
+- /dev/linking_with_flink.html
+- /dev/linking.html
+- /dev/projectsetup/dependencies.html
+- /dev/projectsetup/java_api_quickstart.html
+- /getting-started/project-setup/java_api_quickstart.html
+- /dev/getting-started/project-setup/scala_api_quickstart.html
+- /getting-started/project-setup/scala_api_quickstart.html
+- /quickstart/scala_api_quickstart.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Project Configuration
+
+Every Flink application depends on a set of Flink libraries. At a minimum, the application depends
+on the Flink APIs and, in addition, on certain connector libraries (i.e. Kafka, Cassandra).
+When running Flink applications (either in a distributed deployment or locally for testing),
+the [Flink runtime library](https://ossindex.sonatype.org/component/pkg:maven/org.apache.flink/flink-runtime@1.14.3) 
```

Comment:
This is oddly specific, technically incorrect (you need way more than flink-runtime), it shouldn't contain a hard-coded version, and the link is quite weird (why not maven central?)